src/mbgl/style/sources/vector_source.cpp

```diff
@@ -84,5 +85,11 @@ void VectorSource::loadDescription(FileSource& fileSource) {
     });
 }
 
+bool VectorSource::supportsLayerType(const mbgl::style::LayerTypeInfo* info) const {
+    return !std::strcmp(info->type, "line") || !std::strcmp(info->type, "symbol") ||
```

Comment:
Could these constant strings be fetched from staticTypeInfo of a concrete layer type? Otherwise, could be error prone.

Reply:
Changed with the latest patches.